astropy/table/scripts/showtable.py

```diff
@@ -0,0 +1,111 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+``showtable`` is a command-line script based on ``astropy.io`` and
+``astropy.table`` for printing ASCII, FITS, HDF5 or VOTable files(s) to the
+standard output.
+
+Example usage of ``showtable``:
+
+1. FITS::
+
+    $ showtable astropy/io/fits/tests/data/table.fits
+
+    target V_mag
+    ------- -----
+    NGC1001  11.1
+    NGC1002  12.3
+    NGC1003  15.2
+
+2. ASCII::
+
+    $ showtable astropy/io/ascii/tests/t/simple_csv.csv
+
+     a   b   c
+    --- --- ---
+      1   2   3
+      4   5   6
+
+3. XML::
+
+    $ showtable astropy/io/votable/tests/data/names.xml --max-width 70
+
+               col1             col2     col3  ... col15 col16 col17
+               ---              deg      deg   ...  mag   mag   ---
+    ------------------------- -------- ------- ... ----- ----- -----
+    SSTGLMC G000.0000+00.1611   0.0000  0.1611 ...    --    --    AA
+
+
+
+4. Print all the FITS tables in the current directory::
+
+    $ showtable *.fits
+
+"""
+
+import argparse
+from astropy.table import Table
+from astropy import log
+
+
+def showtable(filename, args):
+    """
+    Read a table and print to the standard output.
+
+    Parameters
+    ----------
+    filename : str
+        The path to a FITS file.
+
+    """
+    read_kwargs = {k: v for k, v in vars(args).items()
+                   if k in ('hdu', 'format', 'table_id') and v is not None}
+    try:
+        table = Table.read(filename, **read_kwargs)
+        formatter = table.more if args.more else table.pprint
+        formatter(max_lines=args.max_lines, max_width=args.max_width,
+                  show_unit=not args.hide_unit, show_dtype=args.show_dtype)
+    except IOError as e:
+        log.error(str(e))
+
+
+def main(args=None):
+    """The main function called by the `showtable` script."""
+    parser = argparse.ArgumentParser(
+        description=('Print tables from ASCII, FITS, HDF5, VOTable file(s).'))
```

Comment:
I'd suggest putting some description here of the default output width / lines and how to control it.  I think the fact that long/wide tables get clipped can be initially confusing.  Something along the lines of:
```
The default behavior is make the table output fit onto a single screen page.  For a
long and wide table this will mean cutting out inner rows and columns.  To print
**all** the rows or columns use ``--max-lines=-1`` or ``max-width=-1``, respectively.
```